pkg/server/server.go

```diff
@@ -19,21 +19,28 @@ import (
 	"github.com/open-sauced/pizza/oven/pkg/providers"
 )
 
+// Config provides the configuration flags set on server startup
+type Config struct {
+	ConfigPath string
+}
```

Comment:
Yes, we'd want a `Config` struct but this would be the actual fields populated from the yaml config file, not just the path to a config on the filesystem.

Reply:
great! at the moment just the repos to never be evicted from cache in the config?